themes/digital.gov/layouts/news/list.html

```diff
@@ -28,63 +28,57 @@ <h2 class="deck">{{- .Params.deck | markdownify -}}</h2>
 
   <section class="stream">
     <div class="grid-container grid-container-desktop">
-      <div class="grid-row">
-
-        <div class="grid-col-12">
-          {{/* Blog Posts by Topic —
-          List all of the blog posts tagged with this topic */}}
-
-          {{/* Gets the past events */}}
-          {{- $events := where .Site.RegularPages.ByDate.Reverse "Section" "events" -}}
-
-          {{- $past_events := where (where .Site.RegularPages.ByDate.Reverse "Section" "events") ".Date.Unix" "<" now.Unix -}}
-
-          {{ $all_events := union $past_events $events }}
-
-          {{/* Gets the recent blog posts */}}
-          {{- $posts := where .Pages "Section" "news" -}}
-
-          {{/* Merges the past events and the recent blog posts */}}
-          {{ $stream := union $all_events $posts }}
-
-          {{/* If there are any items at all... */}}
-          {{- if $stream -}}
-
-            {{/* Loop through all the pages */}}
-            {{/* Also sorting all the items by date and reverse chron */}}
-            {{- range (.Paginate ( $stream.ByDate.Reverse )).Pages -}}
-
-              {{- if eq .Type "events" -}}
-                {{- if ge .Date now.Unix }}
-                  {{- .Render "card-event" -}}
-                {{- else -}}
-                  {{- .Render "card-event-past" -}}
-                {{- end -}}
-              {{- end -}}
-
-              {{- if eq .Type "news" -}}
-                {{/* External links */}}
-                {{- if .Params.source -}}
-                  {{/* see /layouts/news/ for card templates */}}
-                  {{ .Render "card-elsewhere"}}
-                {{ else }}
-                  {{/* Blog Posts — Internal links */}}
-                  {{- if .Params.deck }}
-                    {{/* see /layouts/news/ for card templates */}}
-                    {{ .Render "card-article"}}
-                  {{ else }}
-                    {{/* see /layouts/news/ for card templates */}}
-                    {{ .Render "card-article"}}
-                  {{ end }}
-                {{ end }}
-              {{- end -}}
+      {{/* Blog Posts by Topic —
+      List all of the blog posts tagged with this topic */}}
```

Comment:
Removed unnecessary grid classes `grid-row` and `grid-col-12`. Now it uses its parent container.